src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>7</number>
```

Comment:
You must remember to select the first tab in the Qt Designer plugin before you save a form/dialog as it saves the currently open tab so that is the one that is first is shown when the form is loaded in an application.  {*As it happens I got fed up of this not being done on this form so I think I put in code to override this behaviour in this particular case in the `dlgProfilePreferences` constructor but it is not something that happens normally and can be irritating to the user if a dialogue's opening tag changes unexpectedly between revisions...*} :neckbeard: